packages/sarif-to-markdown/src/sarif-to-markdown.ts

```diff
@@ -1,4 +1,5 @@
-import type { Log, Result, Run } from "sarif";
+// @ts-ignore
```

Comment:
Is @ts-ignore needed?

Reply:
I got errors on my machine
```
Could not find a declaration file for module 'markdown-escape'. '/Users/gabrielmarquet/Desktop/semgrep_fun/security-alert/node_modules/markdown-escape/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/markdown-escape` if it exists or add a new declaration (.d.ts) file containing `declare module 'markdown-escape';`ts(7016)
  ```
  I think it might be missing from dev deps ?

Reply:
By the way you added it 2 years ago 😄  @azu https://github.com/azu/security-alert/blame/75c8d82f52cfca7f39c75e2805b83c83ddaa61c5/packages/sarif-to-markdown/src/sarif-to-markdown.ts#L2

Reply:
This compilation error about markdown-escaope is already ignored by following line
<img width="455" alt="image" src="https://user-images.githubusercontent.com/19714/165958536-4fc3ed58-7c15-42ed-8661-209ba8d4940f.png">

https://github.com/azu/security-alert/pull/23/files#diff-a34c542899b18dbda3d9e5bd01b02093852ac051e7cb09316cc77838410b44e3R3-R4

So, we can just delete Line 1.

Reply:
`// @ts-ignore` means that ignore type error on next line.